src/templates/hi/2019/base.html

```diff
@@ -0,0 +1,192 @@
+{% extends "base/2019/base.html" %}
```

Comment:
Has this changed? We've already merged this already?

Can you do a merge of main into your branch to bring them back in sync?

Reply:
Thanks, did that :)